src/3_api.js

```diff
@@ -290,7 +290,12 @@ Server.prototype.XHRRequest = function(url, data, method, storage, callback, nop
 			}
 			else if (req.status.toString().substring(0, 1) === '4' ||
 					req.status.toString().substring(0, 1) === '5') {
-				callback(new Error('Error in API: ' + req.status), null, req.status);
+				if (req.responseURL && req.responseURL.includes("v2/event")) {
+					callback(req.responseText, null, req.status);
+				}
```

Comment:
Could you add a comment on why the `v2/event` error response is handled differently?